doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -235,25 +264,27 @@ For the User OS, we are using the same `Clear Linux`_ release version as the Ser
 
   .. code-block:: none
 
-     cd ~
-     wget https://download.clearlinux.org/releases/24040/clear/clear-24040-kvm.img.xz
-     unxz clear-24040-kvm.img.xz
+     cd /root/
```

Comment:
You'll need to instruct people to become `root` if you want to download and unzip the image under `/root`